server/entity/splash_potion.go

```diff
@@ -124,24 +135,26 @@ func (s *SplashPotion) Tick(w *world.World, current int64) {
 					splashed.AddEffect(effect.New(eff.Type().(effect.LastingType), eff.Level(), dur))
 				}
 			}
-		} else if s.t == potion.Water() {
-			switch result := result.(type) {
-			case trace.BlockResult:
-				pos := result.BlockPosition().Side(result.Face())
-				if w.Block(pos) == fire() {
-					w.SetBlock(pos, air(), nil)
-				}
-
-				for _, f := range cube.HorizontalFaces() {
-					if h := pos.Side(f); w.Block(h) == fire() {
-						w.SetBlock(h, air(), nil)
-					}
+		}
+		// Splashing Entities and Blocks
+		switch result := result.(type) {
+		case trace.BlockResult:
```

Comment:
A `SplashableEntity` interface was added in this PR but it does not appear to be used in any way. It should probably be implemented here for future use (extinguishing burning players?)

Reply:
Also please clean up the comments in this area; punctuation and capitalization in particular.

Reply:
Maybe, will try to implement it but pretty sure its out of the scope of this pr. I should probably implement it when I implement splash bottles extinguishing players.

Reply:
I wouldn't say so; the interface is implemented here, so might as well support it. To be clear: this does not need to include the functionality for extinguishing burning players.